test/cache/cache.test.js

```diff
@@ -35,7 +35,7 @@ describe('cache', () => {
     });
 
     it('should work in multi compiler mode', () => {
-        rimraf.sync(path.join(__dirname, '../../node_modules/.cache/webpack/cache-test-{first,second}-development'));
+        rimraf.sync(path.join(__dirname, '../../node_modules/.cache/webpack/**'));
```

Comment:
It is unsafe, don't remove full cache, it breaks other tests, in test you need delete only necessary caches

Reply:
How about defining a separate cache-directory for separate tests 😄

Reply:
Can you clarify?

Reply:
`node_modules/.cache/webpack` is the default directory for the cache we can overwrite with `cacheDirectory` option in config.

Reply:
I am already do it on tests level